.github/workflows/ci.yml

```
@@ -66,46 +60,69 @@ jobs:
       - name: "Check out repository code"
         uses: "actions/checkout@v2"
       - name: "Setup environment"
-        uses: "networktocode/gh-action-setup-poetry-environment@v1"
+        uses: "networktocode/gh-action-setup-poetry-environment@v2"
       - name: "Linting: yamllint"
         run: "poetry run invoke yamllint"
-# todo(andy)
-#    needs:
-#      - "black"
-  build:
-    runs-on: "ubuntu-20.04"
-    steps:
-      - name: "Check out repository code"
-        uses: "actions/checkout@v2"
-      - name: "Setup environment"
-        uses: "networktocode/gh-action-setup-poetry-environment@v1"
-      - name: "Copy credentials"
-        run: "cp development/creds.example.env development/creds.env"
-      - name: "Build Container"
-        run: "poetry run invoke build"
-# todo(andy)
-#    needs:
-#      - "bandit"
-#      - "pydocstyle"
-#      - "flake8"
-#      - "yamllint"
   pylint:
+    needs:
+      - "black"
+      - "bandit"
+      - "pydocstyle"
+      - "flake8"
+      - "yamllint"
     runs-on: "ubuntu-20.04"
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.9"]
+        nautobot-version: ["1.2.0-beta.1"]
```

Are we going to have to update this every time we update Nautobot? If so let's go ahead and bump it to v1.2.0 now!
```suggestion
        nautobot-version: ["1.2.0"]
```

---

@jathanism Yes unfortunately we will have to do that. I am not sure if there is an alternative.